ui/src/app/pages/profile/data-user-code-of-conduct.spec.tsx

```diff
@@ -115,14 +48,13 @@ describe('DataUserCodeOfConduct', () => {
     expect(wrapper.find('[data-test-id="submit-ducc-button"]').prop('disabled')).toBeTruthy();
 
     // fill required fields
-    wrapper.find('[data-test-id="dua-initials-input"]').forEach((node, index) => {
+    wrapper.find('[data-test-id="ducc-initials-input"]').forEach((node, index) => {
```

Comment:
Some opportunistic renames in the spirit of RW-4838.  That's a bigger lift, so I'd rather not go too far in that direction in this PR.